lms/static/scripts/frontend_apps/components/JSTORPicker.tsx

```diff
@@ -205,20 +204,17 @@ export default function JSTORPicker({
           </InputGroup>
 
           {metadata.data && (
-            <div
-              className="flex flex-row space-x-2"
+            <UIMessage
+              status={canConfirmSelection ? 'success' : 'info'}
```

Comment:
I'm not clear on why there were nested conditionals here, but in case it matters, I've provided for a neither-success-nor-error-state that won't render an icon, mirroring the existing functionality here.

The pre-existing logic suggested it was possible to have content metadata but that it might not be "selectable" (`canConfirmSelection`), in which case we still want to render the metadata but not the success checkmark next to it. Alert me if this was a misconstrual.